### ISSUE TITLE
useAppPoolCredentials must be documented IIS Configuration Reference

### ISSUE THREAD

**Issue by rossmpersonal** [MicrosoftDocs/iis-docs#521]
useAppPoolCredentials is not documented in IIS Configuration Reference. useAppPoolCredentials needs to be documented in IIS Configuration Reference.

**Comment by lextm** [comment]
Indeed, this property is not documented, https://docs.microsoft.com/en-us/iis/configuration/system.webserver/security/authentication/windowsauthentication/
